challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+	return input.map { |string| string.capitalize }
```

Comment:
That does work! Thanks for letting me know about the more concise syntax... I hadn't seen that before. I appreciate you taking the time to look over my work!